### ISSUE TITLE
Support Node 14

### ISSUE THREAD

**Issue by jmhobbs** [chromaui/chromatic-cli#346]
Some features, like optional chaining, are unsupported in Node 12.  This can be problematic because GitHub actions supports Node 14 (14.16.1 at time of writing), leading to a split where `run` steps are in Node 14, but our action is running on Node 12.

Generally this is not an issue, but can be confusing because a Storybook build will work in a `run` step, then fail when invoked by the action.

This update is blocked by actions/runner#772, as JavaScript actions do not currently support `node14`.  

The current workaround is to build in a `run` step to a known directory, then use `storybookBuildDir` to upload the prebuilt Storybook.

**Comment by jmhobbs** [comment]
cc @romain-blatrix

**Comment by jmhobbs** [comment]
This can also cause issues for compiled packages and bindings, for example node-sass, resulting in errors like:

```
Module build failed (from ./node_modules/sass-extract-loader/index.js):
Error: Missing binding /home/runner/work/<user>/<repository>/node_modules/node-sass/vendor/linux-x64-72/binding.node
Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 12.x

Found bindings for the following environments:
 - Linux 64-bit with Node.js 14.x
 ```

This class of errors can be worked around by using the [setup-node action](https://github.com/actions/setup-node) and using Node 12 in the `run` steps, assuming your code base is compatible with Node 12.

**Comment by ThanakritJuthamongkhonTW** [comment]
Hello @jmhobbs ,

We got the same issue on our ends, would you mind sharing you work around in this thread please?
like your github action workflow file

Thanks in advance

**Comment by jmhobbs** [comment]
For the issue with compiled packages, not missing language features, the action file would look something like this:

```yaml
name: 'Chromatic'

on: push

jobs:
  chromatic-deployment:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v1
      ############################################
      # Insert this step before npm / yarn install
      - uses: actions/setup-node@v2
        with:
          node-version: '12'
      ############################################
      - name: Install dependencies
        run: yarn
      - name: Publish to Chromatic
        uses: chromaui/action@v1
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

**Comment by ThanakritJuthamongkhonTW** [comment]
That is working, many thanks @jmhobbs

**Comment by yshrsmz** [comment]
For me above node-version setting does not work, as package.json has the following engine configuration


```
  "engines": {
    "npm": "please_use_yarn_instead",
    "node": ">=14.15.4"
  },
```

it would be great if you can support node v14

**Comment by ghengeveld** [comment]
This should be fixed with `chromatic@5.8.2`. Please let us know if it doesn't resolve the problem for you.
